SonarQube.TeamBuild.PreProcessor/TeamBuildPreProcessor.cs

```diff
@@ -65,9 +63,11 @@ public TeamBuildPreProcessor(IPreprocessorObjectFactory factory, ILogger logger)
 
         static TeamBuildPreProcessor()
         {
-            plugins = new List<PluginDefinition>();
-            plugins.Add(csharp);
-            plugins.Add(vbnet);
+            plugins = new List<PluginDefinition>
```

Comment:
I have updated VS and it started to report a lot of quality flaws. I fixed one or two but then stopped since it would make this commit unreadale. I will fix that + all other suggestions is a separate commit.

Reply:
@Evangelink When trying to do it, I'm bitten by S3263 saying that I should move this field initializer into a static constructor...

Reply:
@henryju Well the rule check that you do not initialize a static field with the value of another static field that is not yet defined. So it looks like a bug... Could you copy/paste me the line?